libs/ui/src/components/NeoLoading/NeoLoading.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <o-loading
+    :active.sync="isLoadingActive"
+    :can-cancel="canCancel"
+    :overlay-class="'neo-loading-overlay'"
+    :is-full-page="isFullPage">
+    <slot />
+  </o-loading>
+</template>
+
+<script lang="ts" setup>
```

Comment:
use vue.extends. example: https://github.com/kodadot/nft-gallery/blob/main/libs/ui/src/components/NeoCarousel/NeoCarousel.vue

Reply:
I tried doing this, did not work though, maybe model didnt sync or something?

Reply:
aahh I see, let me check

Reply:
oh, yes need to sync. change `v-model` in `Loader.vue` to `:active.sync`

ref: https://vue2.oruga.io/components/Loading.html#props

```
Whether modal is active or not, use the .sync modifier (Vue 2.x) or v-model:active (Vue 3.x) to make it two-way binding
```

Reply:
done the changes